configure.ac

```diff
@@ -1264,11 +1264,24 @@ if $pmda_bpf; then
     libbpf_version_minor=`echo $libbpf_version | cut -d. -f2`
     pmdabpf_modules="biolatency.so runqlat.so"
 
-    AC_MSG_CHECKING([if the bpf PMDA execsnoop module should be included])
+    AC_MSG_CHECKING([if the bpf PMDA modules should be included])
     if test "$libbpf_version_major" -eq 0 -a "$libbpf_version_minor" -lt 7; then
 	AC_MSG_RESULT([no (libbpf version required: 0.7.0, installed: $libbpf_version)])
     else
-	pmdabpf_modules="${pmdabpf_modules} execsnoop.so"
+	pmdabpf_modules="${pmdabpf_modules}\
```

Comment:
All these tools (except CO-RE vfsstat) use `perf_buffer__poll` and this function is part of libbpf 0.7
Test:
```
$ git clone libbpf
$ git checkout v0.6.0
$ ack perf_buffer__poll
```

Reply:
Ok, thanks for checking. Then I only suggest to rename (checking) `if the bpf PMDA modules should be included` to (checking) `if bpf PMDA modules using perf_buffer__poll should be included`, because otherwise it sounds like no module gets included for libbpf < 0.7, which is not true (as we include biolatency and runqlat).